instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -128,6 +128,10 @@ def response_hook(span, instance, response):
 
 
 def _set_connection_attributes(span, conn):
+    if hasattr(conn, "nodes_manager") and hasattr(
+        conn.nodes_manager.default_node, "redis_connection"
+    ):
+        conn = conn.nodes_manager.default_node.redis_connection
```

Comment:
Can we not access `connection_kwargs` directly from the [NodeManager](https://github.com/redis/redis-py/blob/master/redis/cluster.py#L1364) or is this a different set of kwargs?

Reply:
Apologies for the delay, I've been a bit busy with work.

The call to [cleanup_kwargs](https://github.com/redis/redis-py/blob/7d73d742d6df4b3d6f6a1f0ec73e72d4a02add74/redis/cluster.py#L621) removes `host` and `port` from kwargs before instantiating the NodeManager.